distributor-widget/use-cases/connect-to-date-inputs.md

```diff
@@ -0,0 +1,108 @@
+                            event.preventDefault();
+                            // Get the dates from the date form.
+                            const { start, end } = event.target.elements;
+                            const startDate = new Date(start.value);
+                            const endDate = new Date(end.value);
+                            // Use Distributor Widget API to set the dates in Distributor Widget and open it.
+                            dst.setStartDate(startDate);
+                            dst.setEndDate(endDate);
+                            dst.open();
+                        });
+                    };
+
+                    listenOnSubmit();
+
+                    // Enable the submit button, because Distributor Widget is ready to be used.
+                    const enableSubmit = () => {
+                        const submitButton = document.getElementById("dates-submit");
+                        submitButton.value = "Submit";
+                        submitButton.disabled = false;
+                    };
+                    enableSubmit();
+                },
+                // 4. We want to use production data, so we don't pass third configuration parameter with dataBaseUrl. Instead we rely on defaults. But if you want to use environment other than production, you need to set dataBaseUrl here.
+            );
+        </script>
+    </body>
+</html>
+```
+
+### 1. Install Distributor loader script
+
+For more details see [getting started section for installing Distributor loader script](../getting-started.md#install-distributor-loader-script).
+
+### 2. Add form with date inputs
+
+We've added just a simple form with date inputs as an example to give you an idea how to connect your own HTML elements. On your own page you can use any date inputs you want, you just need to connect them with Distributor Widget. Similarly, as later code shows.
+
+The submit button in the form is disabled at page load, so users can't submit the form before the form and Distributor Widget are ready to be used. We enable it later when they're ready.
+
+### 3. Initialize Distributor Widget
+
+#### 3.1 Set configuration id of your Distributor
+
+If you are not sure where to find the configuration id, see [getting started section for initializing Distributor Widget](../getting-started.md#initialize-distributor-widget) for details.
+
+### 4. Optional dataBaseUrl parameter
+
+{% hint style="warning" %}
+By default Distributor Widget uses production environment and data. If this is what you want, you can ignore this section and not set `dataBaseUrl` in third parameter.
+{% endhint %}
+
+But if you want to use Distributor Widget with demo data, you need to set [dataBaseUrl](../reference.md#string-databaseurl). You can see more details in [testing in demo/staging/testing environment section](./testing-in-staging-environment.md).
```

Comment:
I would rephrase it. (including the title)

This guide is written for production environment, if you want to test it during development please refer to our guide how to...